ddev/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * Add color output to tests in CI ([#15774](https://github.com/DataDog/integrations-core/pull/15774))
 
+***Fixed***:
+
+* Fix the datadog_checks_dev tests ([#15779](https://github.com/DataDog/integrations-core/pull/15779))
```

Comment:
This sounds like we just fixed the tests themselves, which isn't something user-facing (changelog-worthy). In fact, iiuc, we fixed the way we invoke tests with our hatch plugin. I'd put that in the changelog.

Reply:
Yes, thanks!!